core/src/test/scala/com/databricks/labs/remorph/transpilers/SnowflakeToDatabricksTranspilerTest.scala

```diff
@@ -9,8 +9,8 @@ class SnowflakeToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTes
   "snowsql commands" should {
 
     "transpile BANG" in {
-      "!set error_flag = true" transpilesTo "--!set error_flag = true;"
-      "!define tablename=CENUSTRACKONE" transpilesTo "--!define tablename=CENUSTRACKONE;"
+      "!set error_flag = true;" transpilesTo "--!set error_flag = true;"
+      "!define tablename=CENUSTRACKONE;" transpilesTo "--!define tablename=CENUSTRACKONE;"
```

Comment:
make format the same on both python and scala side. this PR has to have +3/-3 functional test results

Reply:
I have added +3/-3 test cases for antlr. should i add the same test cases for sqlglot?